win32_build.sh

```diff
@@ -0,0 +1,118 @@
+#!/bin/bash
```

Comment:
Conventions is that `/usr/bin/env bash` is used.

Reply:
Sure, I'll change that.